rumbl/web/static/js/player.js

```diff
@@ -0,0 +1,29 @@
+let Player = {
```

Comment:
@des-des this example is from "The Book" ...
I don't _intend_ to keep the ES6 after I've finished writing out the code. 😉